README.md

```diff
@@ -164,7 +153,45 @@ For more parameters, please see [Monasca Agent Documentation](https://github.com
 
 This will create and run a new service file `/etc/systemd/system/monasca-agent.service` with the configuration set as per the arguments mentioned above. 
 
-### Log agent
+### Uninstalling Metrics agent
+Stop the services: 
+```
+systemctl stop monasca-agent
+```
+Delete created files: 
+
+*(Note: there may be files in /etc/monasca/agent/ that are related to different agents, so only remove the ones listed.)*
+```
+sudo rm -rf /etc/monasca/agent/supervisor.conf
+sudo rm -rf /etc/monasca/agent/agent.yaml
+sudo rm -rf /etc/monasca/agent/conf.d/cpu.yaml
+sudo rm -rf /etc/monasca/agent/conf.d/disk.yaml
+sudo rm -rf /etc/monasca/agent/conf.d/kafka_consumer.yaml 
+sudo rm -rf /etc/monasca/agent/conf.d/load.yaml
+sudo rm -rf /etc/monasca/agent/conf.d/memory.yaml
+sudo rm -rf /etc/monasca/agent/conf.d/network.yaml
+sudo rm -rf /etc/monasca/agent/conf.d/postfix.yaml
+sudo rm -rf /etc/monasca/agent/conf.d/process.yaml
+sudo rm -rf /etc/monasca/agent/conf.d/zk.yaml
+```
+Finally, delete the service related to the agent.
```

Comment:
Remove `mon-agent` user and execute `systemctl daemon-reload` after removing service related file.

Reply:
Done